Makefile

```diff
@@ -1,20 +1,71 @@
 SHELL := /bin/bash
 .SHELLFLAGS = -euo pipefail -c
-MAKEFLAGS = -s
-CGO_ENABLED = 1
+# MAKEFLAGS = -s
```

Comment:
This looks unused. Can we remove that?

Reply:
The -s makes the Make silent.  
I will investigate cgo flag and get back to you.

Reply:
I added code to allow CGO_ENABLED to work either way.

Reply:
I wonder if `MAKEFLAGS` really has an influence here. It is commented out.